scripts/update_icon_list.py

```diff
@@ -1,22 +1,21 @@
 'Update the icon_list module from the FontAwesome GitHub repo'
 import requests
-import yaml
+import json
 import itertools
 import pprint
 
 URI = ('https://raw.githubusercontent.com'
-       '/FortAwesome/Font-Awesome/master/src/icons.yml')
+       '/FortAwesome/Font-Awesome/master/advanced-options/metadata/icons.json')
 
 
 def main():
-    icons_list = yaml.load(requests.get(URI).text)['icons']
-    names_aliases = lambda I: itertools.chain([I['id']],
-                                              I.get('aliases', list()))
-    names = tuple(itertools.chain(*tuple(map(names_aliases, icons_list))))
+    icons_json = requests.get(URI).json()
+    # use only styles
+    icons = {k: icons_json[k]['styles'] for k in icons_json.keys()}
```

Comment:
Cast to `tuple`, since these can be immutable:
``` python
icons = {
    icon_name: tuple(icons_json[icon_name]['styles'])
    for icon_name in icons_json.keys()
}
```
Also, please name your variables descriptively.

Reply:
fixed